pkg/scheduler/listers/listers.go

```diff
@@ -49,3 +50,26 @@ type SharedLister interface {
 	Pods() PodLister
 	NodeInfos() NodeInfoLister
 }
+
+// GetPodServices gets the services that have the selector that match the labels on the given pod.
+func GetPodServices(serviceLister v1listers.ServiceLister, pod *v1.Pod) ([]*v1.Service, error) {
```

Comment:
can we also move the unit tests?

Reply:
Sure, a new file named listers_test.go?

Reply:
Yes please. Can you also add a comment that this should be moved to ServiceAffinity plugin.

Reply:
done

Reply:
@ahg-g sorry to trouble you, could you label an ok-to-test, fixed a bazel error.